recipe/build.sh

```diff
@@ -1,38 +1,19 @@
 #!/bin/bash
 
-args="-DOVITO_BUILD_PLUGIN_MESH=ON \
-      -DOVITO_BUILD_PLUGIN_PARTICLES=ON \
-      -DOVITO_BUILD_PLUGIN_TACHYON=OFF \
-      -DOVITO_BUILD_PLUGIN_CRYSTALANALYSIS=OFF \
-      -DOVITO_BUILD_PLUGIN_NETCDFPLUGIN=OFF \
-      -DOVITO_BUILD_PLUGIN_POVRAY=OFF \
-      -DOVITO_BUILD_PLUGIN_OPENBABELPLUGIN=OFF \
-      -DOVITO_BUILD_PLUGIN_VR=OFF \
-      -DOVITO_BUILD_PLUGIN_CORRELATION=ON \
-      -DOVITO_BUILD_PLUGIN_VOROTOP=ON \
-      -DOVITO_BUILD_GUI=OFF \
-      -DOVITO_BUILD_DOCUMENTATION=OFF \
-      -DCMAKE_BUILD_TYPE=Release \
-      -DCMAKE_INSTALL_PREFIX=${PREFIX}" 
-
 mkdir build
 cd build
-if [ ${PY3K} = 1 ]; then 
-	cmake $args \
-	      -DPYTHON_EXECUTABLE=${PREFIX}/bin/python \
-	      -DPYTHON_INCLUDE_DIR=${PREFIX}/include/python${PY_VER}m \
-	      -DPYTHON_LIBRARY=${PREFIX}/lib/libpython${PY_VER}m${SHLIB_EXT} \
-	      ..
-else 
-	cmake $args \
-	      -DPYTHON_EXECUTABLE=${PREFIX}/bin/python \
-	      -DPYTHON_INCLUDE_DIR=${PREFIX}/include/python${PY_VER} \
-	      -DPYTHON_LIBRARY=${PREFIX}/lib/libpython${PY_VER}${SHLIB_EXT} \
-	      ..
-fi;
-make 
-cp lib/ovito/*${SHLIB_EXT} ${PREFIX}/lib
-cp -r lib/ovito/plugins/python/ovito ${SP_DIR}
-mkdir -p ${PREFIX}/lib/ovito/plugins
-cp lib/ovito/plugins/*${SHLIB_EXT} ${PREFIX}/lib/ovito/plugins
-sed -i 's\/../../..\/../../../../ovito/plugins\g' ${SP_DIR}/ovito/plugins/__init__.py
+
+cmake -DOVITO_BUILD_DOCUMENTATION=ON \
```

Comment:
Btw. Do we need the local documentation? I had the feeling it is just increasing the build size and most people anyway use the online documentation.

Reply:
Good point, I did it for completeness really and without thinking much about the size: indeed the documentation takes 23 MB, an increase from 6MB to 29MB. I don't have strong view on this but I would be inclined to keep it because I always strongly encourage people to read the documentation! And there is a Help/User Manual menu in the GUI with some helps buttons too, and these will most likely not be working.
An alternative would be to several two packages to allow installing ovito without the doc: for example a `ovito-base` without the doc and a `ovito` package with the doc or something along this line.

Reply:
Sure, I guess for a GUI app it makes a lot more sense. I was primarily looking at it from the perspective of using the Python API in a minimal environment like a Docker container. But with switching the versions I guess the GUI use case became the only focus, so let's just keep it.